Makefile

```diff
@@ -65,19 +65,26 @@ build: build-server build-nightly-server build-ad-server build-client \
 build-server: $(BUILDFILE_SERVER)
 .PHONY: build-server
 $(BUILDFILE_SERVER): Makefile $(SERVER_SRC_FILE) $(SERVER_SOURCES)
-	$(BUILD_CMD) \
-		--tag $(SERVER_NAME) --tag $(SERVER_REPO_NAME) \
-		-f $(SERVER_SRC_FILE) $(SERVER_DIR)
-	$(CONTAINER_CMD) inspect -f '{{.Id}}' $(SERVER_NAME) > $(BUILDFILE_SERVER)
+	$(MAKE) _img_build \
```

Comment:
When invoking sub-make, it is always a good idea to call it with explicit directory (`$(MAKE) -C $(DIR)`). After all, someone might call top-level `Makefile` from another directory (something like `make -f samba-container/Makefile`).